<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>phase_coherence_window</class>
 <widget class="QMainWindow" name="phase_coherence_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1601</width>
    <height>981</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="main_layout" stretch="10,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7" stretch="9,1">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item row="0" column="0">
             <widget class="SignalPlot" name="plot_top" native="true"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="vbox_triple_plot"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Log</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="4,1">
           <item>
            <widget class="QPlainTextEdit" name="text_log">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_11">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QPushButton" name="btn_calculate_all">
                 <property name="text">
                  <string>Calculate</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btn_calculate_single">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Transform Single</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QProgressBar" name="progress">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_progress">
               <property name="text">
                <string>TextLabel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,4,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="4,6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Signal X-Limits</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_7" rowstretch="0,0,0,0,0" columnstretch="0,0">
               <item row="3" column="1">
                <widget class="QLineEdit" name="line_xlim2"/>
               </item>
               <item row="4" column="0" colspan="2">
                <widget class="QPushButton" name="btn_refresh">
                 <property name="text">
                  <string>Refresh</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLineEdit" name="line_xlim1"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Select data (scroll area)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="list_select_data"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <widget class="QGroupBox" name="verticalGroupBox_3">
              <property name="title">
               <string>Freq band 2</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QListWidget" name="listwidget_freq_band2"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="verticalGroupBox_2">
              <property name="title">
               <string>Freq band 1</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QListWidget" name="listwidget_freq_band1"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="advanced_options_section" stretch="0,0,0,3,1">
          <item>
           <widget class="QGroupBox" name="horizontalGroupBox_2">
            <property name="title">
             <string>Frequency range 1</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>min:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineedit_freq_range1_min">
               <property name="text">
                <string>0.081</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>max:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineedit_freq_range1_max">
               <property name="text">
                <string>0.3</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="horizontalGroupBox_5">
            <property name="title">
             <string>Frequency range 2</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_16">
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>min:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineedit_freq_range2_min">
               <property name="text">
                <string>0.08</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>max:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineedit_freq_range2_max">
               <property name="text">
                <string>0.31</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Surrogates</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="topMargin">
                <number>5</number>
               </property>
               <item>
                <widget class="QCheckBox" name="checkbox_surr">
                 <property name="text">
                  <string>Enable surrogates</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,4,1">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Surrogates</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="slider_surrogate">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="line_surrogate">
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Parameters</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="3,1">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Window size (s)</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineedit_window_size">
                 <property name="text">
                  <string>50</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,1">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Order (F0)</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineedit_order">
                 <property name="text">
                  <string>2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Overlap (0-1)</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineedit_overlap">
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,1">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Propagation const</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineedit_propagation_const">
                 <property name="text">
                  <string>0.2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="3,1">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Confidence level (%)</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineedit_confidence_level">
                 <property name="text">
                  <string>95</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QPushButton" name="btn_add_paramset">
              <property name="text">
               <string>Add parameter set</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_delete_paramset">
              <property name="text">
               <string>Delete parameter set</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Start: Time evolution: End; Go to start for average</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1601</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignalPlot</class>
   <extends>QWidget</extends>
   <header>gui.plotting.plots.SignalPlot</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
